linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/rule.yml

```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: "Ensure users' .netrc Files are not group or world accessible"
+
+description: |-
+    While the system administrator can establish secure permissions for users' .netrc files, the
+    users can easily override these.
+
+    This rule ensures every file or directory under the home directory related
+    to an interactive user is group-owned by an interactive user.
```

Comment:
Actually, the rule checks and remediate the permission and not the ownership. We could better align this paragraph to the rule content.